odoo/addons/test_lint/__manifest__.py

```diff
@@ -1,11 +1,12 @@
 # -*- coding: utf-8 -*-
 {
-    'name': 'test-eval',
+    'name': 'test-lint',
     'version': '0.1',
     'category': 'Tests',
-    'description': """A module to test Odoo with pylint.""",
+    'description': """A module to test Odoo code with various linters.""",
     'maintainer': 'Odoo SA',
     'depends': ['base'],
     'installable': True,
     'auto_install': False,
```

Comment:
I'm still convinced that's not a good idea. 

As this module cannot be put in "auto_install", developers will never install it. It's way better to provide the right linter config to use inside their editor in order to have direct feedback.

This is clearly the role of runbot to check that code is syntactically correct, businessy correct (and maybe later, [correctly](https://prettier.io/) [formatted](https://black.readthedocs.io))

It should be converted as "steps" on runbot.

//cc @antonylesuisse @odony

Reply:
I agree that it can be discussed and later converted in steps with linter's config files inside the Odoo code but this is needed now to avoid mobile client crashes.
I'm afraid that we will bloat the root dir with dot files for configuring various linters and tools ?!

Reply:
This is required for community proposed PR bugfixes they are not aware of the requirement of the mobile app.